GNUmakefile

```diff
@@ -54,7 +55,8 @@ endif
 ifeq ($(FORCE_DCL),)
   FORCE_DCL=latest
 endif
-terraform build:
+terraform build provider:
+	@make validate_environment;
```

Comment:
what is the benefit of having this be a separate call vs having the same lines directly in here?